File: 3 Terza/Informatica/210511/e1.php
<?php
    echo("Sono nella pagina PHP...ora dovrei fare qualcosa!?<br /><br />");
    $_base = $_REQUEST['base'];
    $_altezza = $_REQUEST['altezza'];

    // step 2 -> disegno il rettangolo
    for($i=1; $i<=$_altezza; $i++){
        for($j=1; $j<=$_base; $j++){
            echo("* ");    
        }
        echo("<br />");
    }
    echo("<br /><br />");
    
    // step 3 -> calcolo e comunicazione dei risultati
    $area = $_base * $_altezza;
    $perimetro = ($_base + $_altezza) * 2;
    echo("Area: ".$area."<br />");
    echo("Perimetro: ".$perimetro."<br />");
?>

File: 3 Terza/Informatica/210511/e1.c
/*
    Semplice algoritmo che richiede le misure di un rettangolo,
    intese come base e altezza, quindi disegna il rettangolo con asterischi
    ed inoltre calcola e comunica area e perimetro.
*/
#include <stdio.h>

int main(){
    int base, altezza;
    //int i, j;
    int area, perimetro;

    // 1° step -> input dei dati.
    printf("Base: ");
    scanf("%d", &base);
    fflush(stdin);
    printf("Altezza: ");
    scanf("%d", &altezza);
    fflush(stdin);

    // 2° step -> rappresento il rettangolo di asterischi.
    for(int i=1; i<=altezza; i++){
        for(int j=1; j<=base; j++){
            printf("* ");
        }
        printf("\n");
    }
    printf("\n\n");

    // 3° step -> calcolo e comunico area e perimetro
    area = base * altezza;
    perimetro = (base + altezza) * 2;
    printf("Area: %d\n", area);
    printf("Perimetro: %d\n", perimetro);
    return(0);
}
